### DISCUSSION TITLE
What core patches are included by default?

### DISCUSSION THREAD

**Discussion by RedPenguin88** [bakkeby/dwm-flexipatch#293] [Q&A] [answered]
The README states:
> Due to the complexity of some of the patches dwm-flexipatch has diverged from mainstream dwm by making some core patches non-optional for maintenance reasons.

Say I were to run the flexipatch-finalizer on a freshly-cloned dwm-flexipatch without any changes to patches.def.h. What non-optional patches would be included by default? How would it differ from a freshly-cloned dwm directly from suckless?

**Comment by bakkeby** [comment, marked as answer]
I suppose the best way to answer this would be to apply flexipatch-finalizer on dwm-flexipatch with no additional patches enabled, then drop in the .git directory from a bare dwm and run `git diff`.

That should show you the scope of the changes. 

In terms of standalone patches we have these integrated:
   - [barmodules](https://github.com/bakkeby/patches/wiki/barmodules/)
   - [tagicons](https://github.com/bakkeby/patches/wiki/tagicons)
   - [urgentborder](https://dwm.suckless.org/patches/urgentborder/)
   - [floatborder](https://dwm.suckless.org/patches/float_border_color/)
   - [titlecolor](https://dwm.suckless.org/patches/titlecolor/)

Branch [dwm-flexipatch-1.0](https://github.com/bakkeby/dwm-flexipatch/tree/dwm-flexipatch-1.0) has a traditional bar and should give you more or less the same as a bare dwm if finalized (less line breaks, empty lines, indentation changes).

**Reply by RedPenguin88** [reply]
Thanks for the thorough reply!
